scaling_performance/optimizing_storage.adoc

```diff
@@ -314,32 +312,65 @@ loopback mounted sparse file. To see if your system is using the loopback mode:
+and the unification process is referred to as a union mount. 
+
+OverlayFS uses one of two graph drivers, *overlay* or *overlay2*. As of Red Hat Enterprise
+Linux 7.2, *overlay*  
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/7.2_release_notes/technology-preview-file_systems[became a supported graph driver]. 
+As of Red Hat Enterprise Linux 7.4, *overlay2* link:https://access.redhat.com/solutions/2908851[became supported]. SELinux on the docker daemon became supported in 
+Red Hat Enterprise Linux 7.4. See the Red Hat Enterprise Linux release notes for information on using OverlayFS with your version of RHEL, including supportability and usage caveats.
+
+The *overlay2* driver natively supports up to 128 lower OverlayFS layers, where,
+the *overlay* driver works only with a single lower OverlayFS layer. Because of this capability, the *overlay2* driver provides better performance 
+for layer-related Docker commands such as `docker build`, and consumes fewer inodes on the backing filesystem. 
+
+Because the *overlay* driver works only with a single lower OverlayFS layer, multi-layered images cannot be implemented as multiple OverlayFS layers. 
+Instead, each image layer is implemented as its own directory under *_/var/lib/docker/overlay_*. 
+Hard links are then used as a space-efficient way to reference data shared with lower layers.
+
+Docker link:https://docs.docker.com/storage/storagedriver/overlayfs-driver/[recommends] using the *overlay2* driver with OverlayFS rather than 
+the *overlay* driver, because it is more efficient in terms of inode utilization.
+
+[NOTE]
+====
+To use overlay2 with RHEL or CentOS you need version 3.10.0-693 or higher of the kernel.
+====
+
+[[benefits-of-using-the-overlay-graph-driver]]
+=== Benefits of Using the OverlayFS with SELinux 
+
+The main advantage of the OverlayFS graph is Linux page cache sharing among
+containers sharing an image on the same node. This attribute of OverlayFS leads to
 reduced input/output (I/O) during container startup (and, thus, faster container
 startup time by several hundred milliseconds), as well as reduced memory usage
 when similar images are running on a node. Both of these results are beneficial
 in many environments, especially those with the goal of optimizing for density
 and have high container churn rate (such as a build farm), or those that have
 significant overlap in image content.
 
-Page cache sharing is not possible with `DeviceMapper` because thin-provisioned
+Page cache sharing is not possible with DeviceMapper because thin-provisioned
 devices are allocated on a per-container basis.
+
+[NOTE]
+====
+DeviceMapper is the default Docker storage configuration on Red Hat Enterprise Linux. 
+The use of OverlayFS as the container storage
+technology is under evaluation and moving Red Hat Enterprise Linux to OverlayFS as
```

Comment:
Do we usually discuss potential changes like this in doc?

Reply:
@kalexand-rh Yeah, we do (I am not a fan, but have never fought it)

Reply:
@mburke5678, thanks for the confirmation. (I'm not a fan either. I wonder who we'd have to go fight.)